### PR TITLE
Create user entity

### PR DESCRIPTION
**Pull request by IdaFagerlund**

### CODE REVIEW

File: src/main/java/com/example/demo/entities/User.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
+package com.example.demo.entities;
+
+import javax.persistence.*;
+
+@Entity
+@Table
+public class User {
```
Comment (DaniellaNoren) [inline src/main/java/com/example/demo/entities/User.java:7]: Why the fuck did you do this!?????+1?!?!?!??!?!?!?

File: src/main/java/com/example/demo/entities/User.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
```
Comment (DaniellaNoren) [inline src/main/java/com/example/demo/entities/User.java:10]: Oh no!!!

**Review by DaniellaNoren — CHANGES_REQUESTED** [review]
what the fuuuck

File: src/main/java/com/example/demo/entities/User.java (hunk 2 above)
Comment (IdaFagerlund) [reply src/main/java/com/example/demo/entities/User.java:10]: fixed!

**Review by joeliths — APPROVED** [review]

**Review by PontusF — APPROVED** [review]

**Review by joeliths — APPROVED** [review]

**Pull request merged**